svtools/vcf/variant.py

```diff
@@ -118,5 +122,21 @@ def get_var_string(self):
                 self.get_format_string(),
                 '\t'.join(self.genotype(s).get_gt_string() for s in self.sample_list)
             ]))
+        else:
+            if len(self.gts_string) == 0:
```

Comment:
If you set gts_string to None then this could simply be if gts_string == None rather than calculating the length.